pkg/engine/variables/operator/allin.go

```diff
@@ -41,7 +44,7 @@ func (allin AllInHandler) Evaluate(key, value interface{}) bool {
 }
 
 func (allin AllInHandler) validateValueWithStringPattern(key string, value interface{}) (keyExists bool) {
-	invalidType, keyExists := keyExistsInArray(key, value, allin.log)
```

Comment:
Is `keyExistsInArray` still needed? Should it be deleted?

Reply:
Hey @MarcelMue `keyExistsInArray` is defined in `in.go` and is required for `in` and `notIn` operators, it is deprecated now, in the next release it will be removed and then these would be released too.